website/scripts/build-html-email.js

```diff
@@ -0,0 +1,104 @@
+module.exports = {
+
+
+  friendlyName: 'Build HTML email',
+
+
+  description: 'Generate an HTML partial for the Fleet newsletter from Markdown files in the articles/ folder of the fleetdm/fleet repo.',
+
+
+  inputs: {
+    articleFilename: {type: 'string', description: 'The filename of the article that will be converted into an HTML email partial', required: true},
+  },
+
+
+  fn: async function ({ articleFilename }) {
+
+    let path = require('path');
+    let YAML = require('yaml');
+
+    let topLvlRepoPath = path.resolve(sails.config.appPath, '../');
+
+    let APP_PATH_TO_COMPILED_EMAIL_PARTIALS = 'views/emails/newsletter-partials';
+
+
+    let extensionedArticleFilename = articleFilename;
+
+    // Since this script only handles Markdown files in the articles/ folders, we'll make the file extension optional.
+    if(!_.endsWith(articleFilename, '.md')) {
+      // If the file was specified without a file extension, we'll add `.md` to the provided filename, and log a warning.
+      extensionedArticleFilename = extensionedArticleFilename + '.md';
+      sails.log.warn('The filename provided is missing the .md file extension, appending `.md` to the provided articleFilename: '+articleFilename)
+    }
+
+    // Get the filename without the .md file extension. This will be used to build the final filename
+    let unextensionedArticleFilename = _.trimRight(extensionedArticleFilename, '.md');
+
+    // Build the filename for the final HTML partial.
+    let extensionedFileNameForEmailPartial = unextensionedArticleFilename.replace(/\./g, '-')+'.ejs';
```

Comment:
Prefix with `email-` for consistency

Probably also add `article` in the fileanme somewhere to clarify what kind of email this is.

So maybe:

```js
let extensionedFileNameForEmailPartial = 'email-article-'+unextensionedArticleFilename.replace(/\./g, '-')+'.ejs';
```